db/schema.rb

```diff
@@ -184,6 +184,7 @@
     t.boolean "high_pupil_premium", default: false, null: false
     t.text "postcode_without_spaces"
     t.integer "number_of_pupils"
+    t.boolean "eyl_eligible", default: false
```

Comment:
What do you think about including something related to funding in the name of this column? Something like `eyl_funding_eligible`. That'd make it very explicit that it's funding related rather than what courses you can apply for.

Reply:
Indeed that should provide better context. I've pushed the changes